### ISSUE TITLE
[MAS42.4.3][Keyboard Navigation -Open bot dialog] Focus loss occurred after browse control which is present in the open bot dialog 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1723]
**Actual Result:**
Focus loss occurred after browse control which is present in the open bot dialog  

**Expected Result:**
Focus should not loss after browse control focus should moves to the next control i.e Microsoft soft App ID text field. 

**Repro Steps:**
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the Welcome tab screen
3.	Navigate to the open bot control 
4.	Open bot dialog gets opened and Navigate to the browse control using tab key 
5.	Observe the issue

**User Impact:**
Users who rely on keyboard navigation will be affected if the keyboard focus gest loss on control  so that keyboard users unable to get where the keyboard focus is lands on it.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)
[MAS2.4.3_Open Bot dialog.zip](https://github.com/microsoft/BotFramework-Emulator/files/3500560/MAS2.4.3_Open.Bot.dialog.zip)
